drivers/lte_link_control/lte_lc.c

```diff
@@ -106,6 +113,22 @@ static int w_lte_lc_init_and_connect(struct device *unused)
 		close(at_socket_fd);
 		return err;
 	}
+#endif
+#if defined(CONFIG_LTE_LEGACY_PCO_MODE)
+	err = at_cmd(at_socket_fd, legacy_pco, AT_CMD_SIZE(legacy_pco));
+	if (err) {
+		close(at_socket_fd);
+		return err;
+	}
+	LOG_INF("Using legacy LTE PCO mode...\n");
```

Comment:
The `\n` is redundant here, as logger module adds a newline as well, same other places in the file

Reply:
@joakimtoe Let's fix this and merge it.

Reply:
@lemrey done!